### DISCUSSION TITLE
Does a low `unspliced` proportion have an impact on `scvelo` analysis?

### DISCUSSION THREAD

**Discussion by jiangpuxuan** [theislab/scvelo#866] [Q&A] [answered]
Hi, I have many germ cell scRNA datasets, and all of them have a very low  `unspliced` proportion ( about 5% to 10% ). 
The `scvelo+cellrank` pipeline gave me a wierd result of `terminal/initial state` that is contrary to Prior Knowledge.
So, I wonder if a low `unspliced` proportion have an impact on `scvelo` analysis. 
Thank you very much.

**Comment by WeilerP** [comment, marked as answer]
@jiangpuxuan, yes, it can (and most likely does) have an impact. Please have a look at related issues and discussions. My best guess guess is that something similar to #216 or #462 is happening.
